src/parser/transform/statement/transform_show_select.cpp

```diff
@@ -0,0 +1,24 @@
+#include "duckdb/parser/statement/show_statement.hpp"
+#include "duckdb/parser/sql_statement.hpp"
+#include "duckdb/parser/transformer.hpp"
+
+using namespace duckdb;
+using namespace std;
+
+unique_ptr<ShowStatement> Transformer::TransformShowSelect(PGNode *node) {
+
+	// we capture the select statement of SHOW
+	auto stmt = reinterpret_cast<PGVariableShowSelectStmt *>(node);
+	auto select_stmt = reinterpret_cast<PGSelectStmt *>(stmt->stmt);
+
+	if(string(stmt->name) == "select"){
```

Comment:
Any reason this comparison is here? The function has no return statement if the comparison is false.

Reply:
If the name is not equal to "select", it could be equal to "tables" or the name of a table, which will then be handled as a pragma statement. So, when will this exception be thrown? Because, if I'm not missing something here, TransformShowSelect is only called when there is a select statement after SHOW.

Reply:
I think so too; I believe this check might not be required at all, right? If I do e.g. `SHOW tbl` then this will not be called at all. Perhaps the entire check can be removed?

Also, could you perhaps add a test that verifies that this still works:

```sql
create table "select"(i integer);
describe "select";
--- i,INTEGER,YES,NULL,NULL,NULL
```

Reply:
I think so, too. I removed it and added the test case in "test/sql/pragma/test_show_tables.test"